packages/row/src/react/index.js

```diff
@@ -96,22 +96,19 @@ const ActionBar = props => (
 )
 
 const ActionBarAction = withTheme(
-  class extends React.PureComponent {
-    render() {
-      // eslint-disable-next-line react/prop-types
-      const { icon, ...props } = this.props
-      const filteredProps = filterReactProps(props, {
-        tagName: 'button'
-      })
-
-      return (
-        <button {...styles.actionBarAction(props)} {...filteredProps}>
-          {icon}
-        </button>
-      )
-    }
-  }
+  React.forwardRef((props, ref) => {
+    // eslint-disable-next-line react/prop-types
+    const { icon, ...rest } = props
+    const filteredProps = filterReactProps(rest, { tagName: 'button' })
+
+    return (
+      <button {...styles.actionBarAction(props)} ref={ref} {...filteredProps}>
```

Comment:
this was all so they could get the Action button's ref for positioning action menus, tooltips, or what-not, right?

Reply:
mostly so will could add a ref to the Row.Action...but yes we need greater ref support